spec/support/shared_examples/remote_duplicate_support.rb

```diff
@@ -41,7 +48,7 @@
       end
 
       def make_request
-        post :remote_duplicate, id: resource.id, user_email: user_email, add_to_portal: portal_url
+        post :remote_duplicate, id: resource.id, user_email: user_email, add_to_portal: portal_url, author_url: author_url
```

Comment:
This should only send the author_url when the home controller is being tested.
I'd suggest removing `let(:author_url)` from this file. 
And moving the `let(:aruthor_url)...` up into `it_behaves_like "remote duplicate support" do` block in the home_controller_spec.

I think if the author_url is undefined or nil then this line in make_request won't send the parameter. But you might need to change it so it author_url key isn't even in the hash send to the post method.